src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/DataprocDAO.scala

```diff
@@ -30,4 +29,6 @@ trait DataprocDAO {
   def getClusterErrorDetails(operationName: OperationName)(implicit executionContext: ExecutionContext): Future[Option[ClusterErrorDetails]]
 
   def deleteBucket(googleProject: GoogleProject, gcsBucketName: GcsBucketName)(implicit executionContext: ExecutionContext): Future[Unit]
+
+  def getEmailFromAccessToken(accessToken: String)(implicit executionContext: ExecutionContext): Future[String]
```

Comment:
Not really a Dataproc operation. :) But I guess this is ok given that we have a future housekeeping task for breaking up `DataprocDAO`.

Reply:
Also in the spirit of better types this could return a `WorkbenchUserEmail` (which is a case class in workbench-libs) instead of a String.